sdk/go/Makefile

```diff
@@ -27,19 +27,21 @@ build-examples: $(EXAMPLES_DIR)/http-tinygo/main.wasm
 build-examples: $(EXAMPLES_DIR)/tinygo-outbound-redis/main.wasm
 build-examples: $(EXAMPLES_DIR)/tinygo-redis/main.wasm
 build-examples: $(EXAMPLES_DIR)/tinygo-key-value/main.wasm
+build-examples: $(EXAMPLES_DIR)/tinygo-outbound-pg/main.wasm
 
 $(EXAMPLES_DIR)/%/main.wasm: $(EXAMPLES_DIR)/%/main.go
 	tinygo build -target=wasi -gc=leaking -no-debug -o $@ $<
 
 # ----------------------------------------------------------------------
 # Generate C bindings
 # ----------------------------------------------------------------------
-GENERATED_SPIN_CONFIG    = config/spin-config.c config/spin-config.h
-GENERATED_OUTBOUND_HTTP  = http/wasi-outbound-http.c http/wasi-outbound-http.h
-GENERATED_SPIN_HTTP      = http/spin-http.c http/spin-http.h
-GENERATED_OUTBOUND_REDIS = redis/outbound-redis.c redis/outbound-redis.h
-GENERATED_SPIN_REDIS     = redis/spin-redis.c redis/spin-redis.h
-GENERATED_KEY_VALUE      = key_value/key-value.c key_value/key-value.h
+GENERATED_SPIN_CONFIG     = config/spin-config.c config/spin-config.h
+GENERATED_OUTBOUND_HTTP   = http/wasi-outbound-http.c http/wasi-outbound-http.h
+GENERATED_SPIN_HTTP       = http/spin-http.c http/spin-http.h
+GENERATED_OUTBOUND_REDIS  = redis/outbound-redis.c redis/outbound-redis.h
+GENERATED_SPIN_REDIS      = redis/spin-redis.c redis/spin-redis.h
+GENERATED_KEY_VALUE       = key_value/key-value.c key_value/key-value.h
+GENERATED_OUTBOUND_PG     = postgres/outbound-pg.c postgres/outbound-pg.h
```

Comment:
Would you please add `postgres/sdk-version-go.c` to `SDK_VERSION_DEST_FILES` below?  That helps ensure the SDK version information is embedded in components built with the SDK.

Reply:
Will do, I need to rebase onto v1.0 branch first and then I'll add it to avoid any merge conflicts.